js/src/ui/Form/AddressSelect/addressSelect.js

```diff
@@ -613,6 +617,10 @@ class AddressSelect extends Component {
       focusedItem: null,
       inputValue: value
     });
+
+    if (apiutil.isAddressValid(value)) {
```

Comment:
Why not importing directly `isAddressValid` (I think that's how it's done pretty much everywhere)